tests/testthat/_snaps/dm_to_tibble.md

```diff
@@ -0,0 +1,458 @@
+    Code
+      print(serialized_from_tf_1$tf_2_json_child)
+    Output
+       [1] "{\"empty\":[true],\"data\":[{\"c\":\"a\",\"e\":\"a\",\"e1\":1,\"tf_3_json_parent\":\"{\\\"data\\\":[{}]}\"}]}"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
+       [2] "{\"empty\":{},\"data\":[{\"c\":\"elephant\",\"e\":\"D\",\"e1\":4,\"tf_3_json_parent\":\"{\\\"data\\\":[{\\\"g\\\":\\\"three\\\",\\\"tf_4_json_child\\\":\\\"{\\\\\\\"empty\\\\\\\":{},\\\\\\\"data\\\\\\\":[{\\\\\\\"h\\\\\\\":\\\\\\\"b\\\\\\\",\\\\\\\"i\\\\\\\":\\\\\\\"four\\\\\\\",\\\\\\\"tf_5_json_child\\\\\\\":\\\\\\\"{\\\\\\\\\\\\\\\"empty\\\\\\\\\\\\\\\":{},\\\\\\\\\\\\\\\"data\\\\\\\\\\\\\\\":[{\\\\\\\\\\\\\\\"k\\\\\\\\\\\\\\\":1,\\\\\\\\\\\\\\\"m\\\\\\\\\\\\\\\":\\\\\\\\\\\\\\\"house\\\\\\\\\\\\\\\",\\\\\\\\\\\\\\\"tf_6_json_parent\\\\\\\\\\\\\\\":\\\\\\\\\\\\\\\"{\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\"data\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\":[{\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\"o\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\":\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\"e\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\"}]}\\\\\\\\\\\\\\\"}]}\\\\\\\"}]}\\\"}]}\"}]}"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  
```

Comment:
This looks like too many backslashes to me. Is this JSON packed inside JSON?

Reply:
Yes it is, so columns can be unwrapped recursively from SQL if needed. Isn't it what we want?